wled00/colors.cpp

```diff
@@ -70,27 +69,56 @@ uint32_t color_add(uint32_t c1, uint32_t c2, bool fast)
 
 uint32_t color_fade(uint32_t c1, uint8_t amount, bool video)
 {
-  if (c1 == BLACK || amount + video == 0) return BLACK;
+  if (c1 == BLACK || amount == 0) return BLACK;
+  else if (amount == 255) return c1;  
```

Comment:
`else` really does no difference here.

Reply:
true, I had this removed but I shifted code around so much, that probably got lost...